Makefile

```diff
@@ -19,3 +19,12 @@ lint: prometheus_alert_rules.yaml
 
 clean:
 	rm -rf prometheus_alert_rules.yaml
+
+generate_k8s:
+	cd extras; cp manifests/prometheus-rules.yaml .; jb install; ./build.sh example.jsonnet; cd -
```

Comment:
I think line 25 will always fail because `jb install;` will always clone the upstream. The genterated  manifests/prometheus-rules.yaml from `./build.sh example.jsonnet;` will always be made from the upstream cloned code. Because of this comparision will always fail. 
I think before `./build.sh example.jsonnet;` the following line should be added so that the cloned code gets the changes made in the pr : `cp -R ./../alerts/* ./vendor/ceph-mixins/alerts/`

Reply:
Yes while verifying the changes I figured this out. Will test ad send the updated PR soon.

Reply:
Exactly: And this should copy all the files that the dev can change. Eg config files, alert rule files, etc.

Reply:
This PR we will take up later as there are other issues while copying the files under `vendor` directory like cyclic dependency.